File: 04.CLICON/EurekaBank_SOAP_CliCon/src/ec/edu/monster/modelo/PartidoFutbol.java
package ec.edu.monster.modelo;

import java.util.Date;

public class PartidoFutbol {
    private String codigo;
    private String equipoLocal;
    private String equipoVisitante;
    private Date fecha;
    private String lugar;

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getEquipoLocal() {
        return equipoLocal;
    }

    public void setEquipoLocal(String equipoLocal) {
        this.equipoLocal = equipoLocal;
    }

    public String getEquipoVisitante() {
        return equipoVisitante;
    }

    public void setEquipoVisitante(String equipoVisitante) {
        this.equipoVisitante = equipoVisitante;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }

    public String getLugar() {
        return lugar;
    }

    public void setLugar(String lugar) {
        this.lugar = lugar;
    }
}

File: 04.CLIMOV/app/src/main/java/ec/edu/monster/model/PartidoFutbol.kt
package ec.edu.monster.model

import java.util.Date

data class PartidoFutbol(
    var codigo: String = "",
    var equipoLocal: String = "",
    var equipoVisitante: String = "",
    var fecha: Date = Date(),
    var lugar: String = ""
) {
    val descripcion: String
        get() = "$equipoLocal vs $equipoVisitante"
}
